packages/story-editor/src/elements/video/playPauseButton.js

```diff
@@ -209,6 +210,35 @@ function PlayPauseButton({
     };
   }, [checkMouseInBBox]);
 
+  const handleKeyDown = useCallback(
+    ({ key }) => {
+      if (allowKeyboardSupport && key === ' ') {
```

Comment:
Since `useKeyDownEffect` below is already handling the `Space` part, there's no need to check for the key here.